client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-expanded-content/in-progress-icon.tsx

```diff
@@ -0,0 +1,26 @@
+import { Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import Tooltip from 'calypso/components/tooltip';
+
+export default function InProgressIcon( { id }: { id: string } ) {
```

Comment:
Is there a reason why we need the ID props here? Maybe we can drop this? wdyt?

Reply:
I was under the impression that `id` was mandatory. It works fine without `id` too. Fixed here: https://github.com/Automattic/wp-calypso/pull/82091/commits/0313769c2fc14c98d69c74ff3d9425b87cfd80af